src/Uno.UI/UI/Xaml/AdaptiveTrigger.cs

```diff
@@ -22,25 +22,22 @@ private void UpdateState()
 		{
 			var size = Window.Current.Bounds;
 
-			var isMinWidthSet = MinWindowWidth != -1;
```

Comment:
this check isn't necessary?

Reply:
Nope ;-)

1. It was comparing against scalar value, which is risky when using a floated point value
2. The check was useless in this circonstances and the result is the same.

Reply:
more tests has been added to check those cases.